README.md

```diff
@@ -955,38 +955,78 @@ An instance `engine` of a `tnt.Engine()` implements two main methods:
   * `engine:test()`,  for evaluating a model on data
         (optionally with respect to a `nn.Criterion`).
 
-The Engine can be implemented for any common underlying training and testing
+The `Engine` can be implemented for any common underlying training and testing
 procedure involving a model and data. It can also be designed to allow user
 control after certain events such as forward prop, criterion evaluation, or the
 end of an epoch, by using coroutines (see `tnt.SGDEngine`).
 
 
 ### tnt.SGDEngine
 
-The SGDEngine module implements the Stochastic Gradient Descent training
+The `SGDEngine` module implements the Stochastic Gradient Descent training
 procedure in `train`, including data sampling, forward prop, back prop, and
 parameter updates. It also operates as a coroutine allowing a user control
  (i.e. increment some sort of `tnt.Meter`) at events such as 'start',
 'start-epoch', 'forward', 'forward-criterion', 'backward', etc.
+The available hooks are the following:
```

Comment:
Actually, the hooks that are available are `Engine`-specific. Something like a `SVMEngine` (if we ever write it) may have very different hooks.